### PR TITLE
docs: add ways-to-contribute.md

### LINKED ISSUE #25: Create document to be clear about multiple way newcomers can contribute
## Description

List in a document the many ways people can contribute to our community. Take inspiration from OS Ambassador's document. This is not a blog post, just a list of examples in bullet points, so this is easy to share with newcomers in our community.

## Acceptance criteria

- [ ] Mention how to contribute to many categories we encourage in the community
- [ ] Link to documents within this repository
- [ ] Mention the Calendar from the community

### PR DESCRIPTION
**Pull request by isabelcosta**
## Description

Add ways to contribute documentation. 
This is meant to be an initial version, I am not looking for a perfect document at the moment.

Fixes #25 

## Why

This will be helpful to answer a common question from the community, on how to contribute 🎉 
We can link this document to newcomers, so they know all the types of contributions we welcome and help them get started faster!

### CODE REVIEW

**Review by gaurivn — APPROVED** [review]
Well written looks perfect to me

**Comment by isabelcosta** [comment]
I'll merge this since at the moment we have 1 minimum approval requirement and @gaurivn already approved it 🙌 Thank you so much!

**Comment by gaurivn** [comment]
You're welcome :smiley:

File: ways-to-contribute.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Quality Assurance
+
+Help us improve the code and overall quality of our apps. Zulip stream: [#quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance).
+
+- Verify bugs reported in project issues by reproducing them;
+- Test pull request code and report the results, so we can verify new code is working properly. Learn more about our processs here at [quality-assurance.md](/quality-assurance.md);
+- Help us document test cases for our apps;
+- Test our apps in production and report bugs through creating issues;
+- Improve our code coverage by writing tests (unit and UI tests).
```
Comment (rpattath) [inline ways-to-contribute.md:26]: Maybe unit, integration and UI tests.

**Pull request merged**